### ISSUE TITLE
Issue with overwriting highlights

### ISSUE THREAD

**Issue by mbigus** [BrandonBlaschke/vscode-easy-highlight#18]
If you highlight some text - then highlight it again - or even a few more times - the highlight color keeps getting darker which is fine - but then the only problem is you can't remove the highlight - even if you use remove all......

**Comment by BachiMjavanadze** [comment]
please fix it

**Comment by BrandonBlaschke** [comment]
Yeah agreed it should be able to be removed. Updated so it does not allow duplicate highlights on the same range. You can apply a different range that overlaps and it will darken the color if needed. Can also change the color if you update the highlight color and re-apply. Thank you for the issue.

PR: https://github.com/BrandonBlaschke/vscode-easy-highlight/pull/31
